runtime/src/bank.rs

```diff
@@ -6540,6 +6548,12 @@ impl Bank {
                 vote_cost_limit,
             );
         }
+
+        if new_feature_activations.contains(&feature_set::remove_accounts_delta_hash::id()) {
+            // If the accounts delta hash has been removed, then we no longer need to compute the
+            // AccountHash for modified accounts, and can stop the background account hasher.
+            self.rc.accounts.accounts_db.stop_background_hasher();
+        }
```

Comment:
these chunks seem to be modifying behavior behind a feature gate. the feature gate must be rekeyed whenever this happens

Reply:
I'm comfortable with no rekey given ☝️ 

Clearly the "When to rekey" rules need to be agreed upon and documented. I'll take a stab at that next week

Reply:
why is perf stuff behind a feature gate?

first release is only relevant if the pr land before the release is cut. making a special case for this situation is asking for mistakes. keypairs are free

Reply:
sigh... can we try to avoid architectures like this? so messy on so many levels